Greve/template/README.md

```diff
@@ -1,13 +1 @@
-Ok so I guess you are reading this cuz you wanna use my code. There are some
-functions that do stuf and thats:
-
-    >>> from simple_functions import factorial
-    >>> factorial(10)
-    9
-
-and this other part does something.  I forget why that I did it:
-
-    >>> fibonnaccci(100)
-    [0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89]
-
-If you can't use it, its kind of your problem, not mine!
+This repository contains a few useful functions.
```

Comment:
Maybe it is better to list these functions and describe their purpose and arguments.

Reply:
You are right. I was being lazy. Let me do that know and make these changes to my pull request. Cheers - T

Reply:
Happy now? :-)